e2e/fixtures/assets/scan_help

```diff
@@ -51,7 +51,7 @@ Flags:
   -r, --secrets-regexes-path string   path to secrets regex rules configuration file
       --timeout int                   number of seconds the query has to execute before being canceled (default 60)
   -t, --type strings                  case insensitive list of platform types to scan
-                                      (Ansible, AzureResourceManager, Buildah, CloudFormation, Dockerfile, GRPC, GoogleDeploymentManager, Kubernetes, OpenAPI, Terraform)
+                                      (Ansible, AzureResourceManager, Buildah, CloudFormation, DockerCompose, Dockerfile, GRPC, GoogleDeploymentManager, Kubernetes, OpenAPI, Terraform)
```

Comment:
Can you also update in `kics/docs/dockerhub.md` and `kics/docs/commands.md`, please?

In addition to that, can you add DockerCompose to `kics/docs/platforms.md`, please?